homeworks/nikita.karchinskiy_Nikkarchinskiy/6-typescript/dist/style.css

```diff
@@ -0,0 +1,51 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.main-wrapper {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  height: 700px;
+  gap: 50px;
+  padding-top: 20px;
+}
+
+.circle-wrapper {
+  display: flex;
+  align-items: stretch;
+  justify-content: center;
+  width: max-content;
```

Comment:
Do you need here `max-content`? It's better not to use it due to limited support in browsers. https://caniuse.com/?search=max-content

Try to remade CSS